fairer-iOS/fairer-iOS/Screens/HomeViewController.swift

```diff
@@ -246,25 +316,48 @@ final class HomeViewController: BaseViewController {
         // FIXME: - 집안일 추가 뷰로 연결
         print("tap")
     }
-}
-
-    // MARK: - protocol
-
-protocol CollectionViewHeightProtocol: AnyObject {
-    func getCollectionViewHeight(cellNum: Int)
-}
-
-    // MARK: - extension
-
-extension HomeViewController: CollectionViewHeightProtocol {
-    func getCollectionViewHeight(cellNum: Int) {
-        cellHeight = CGFloat(cellNum) * SizeLiteral.homeViewWorkCellHeight
-        calendarDailyCollecionView.snp.updateConstraints {
-            $0.height.equalTo(cellHeight)
+    
+    @objc func handleSwipes(_ sender:UISwipeGestureRecognizer) {
+        if (sender.direction == .left) {
+            self.homeWeekCalendarCollectionView.getAfterWeekDate()
+        }
+            
+        if (sender.direction == .right) {
+            self.homeWeekCalendarCollectionView.getBeforeWeekDate()
+        }
+    }
+    
+    private func moveToTodayDate() {
+        self.homeWeekCalendarCollectionView.datePickedByOthers = ""
+        self.homeCalenderView.calendarMonthLabelButton.setTitle("\(Date().yearToString)년 \(Date().monthToString)월", for: .normal)
+        self.homeWeekCalendarCollectionView.startOfWeekDate = Date().startOfWeek
+        self.homeWeekCalendarCollectionView.dateList = self.homeWeekCalendarCollectionView.getThisWeekInInt()
+        self.homeWeekCalendarCollectionView.fullDateList = self.homeWeekCalendarCollectionView.getThisWeekInDate()
+        self.homeWeekCalendarCollectionView.collectionView.reloadData()
+    }
+    
+    private func moveToDatePicker() {
+        self.homeWeekCalendarCollectionView.datePickedByOthers = ""
+        self.datePickerView.isHidden = false
+        self.setupAlphaNavigationBar()
+    }
+    
+    private func setCalendarHight() {
```

Comment:
p5;
```suggestion
    private func setCalendarHeight() {
```
height.......죄송합니다......

Reply:
[36e59b1](https://github.com/fairer-iOS/fairer-iOS/pull/79/commits/36e59b13fe0e770849dc525c34ffde2201d6b14c)